CMake/DisplayCluster.cmake

```diff
@@ -13,8 +13,11 @@ set(DISPLAYCLUSTER_DEB_DEPENDS libavutil-dev libavformat-dev libavcodec-dev
   libopenmpi-dev openmpi-bin libswscale-dev libxmu-dev libpoppler-qt5-dev
   libboost-date-time-dev libboost-serialization-dev libboost-test-dev
   libboost-program-options-dev libboost-regex-dev libboost-system-dev
-  libboost-thread-dev libfcgi-dev)
-set(DISPLAYCLUSTER_PORT_DEPENDS ffmpeg boost poppler fcgi)
+  libboost-thread-dev libfcgi-dev qt5-default libqt5core5 libqt5declarative5
```

Comment:
qt5-default is not necessary imo and has potential side effects for other developments as it changes the system's default version used for any compilation.